src/main/resources/hudson/matrix/LabelAxis/help-labels_nl.html

```diff
@@ -1,15 +1,24 @@
 <div>
-  Geef de nodes, waar uw bouwpoging uitgevoerd wordt, op.
+  Geef de nodes op waar uw bouwpoging uitgevoerd zal worden.
```

Comment:
Unrelated I suppose but fine. (In the longer term, we would like to split translations into dedicated plugins, as has already begun for Simplified Chinese IIRC.)